invenio_cli/helpers/package_managers.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2025 TU Wien.
+#
+# Invenio-Cli is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Wrappers around various package managers to be used under the hood."""
+
+import os
+from abc import ABC
+from typing import List
```

Comment:
if you prefer, we can strip those type hints out again. i just added them for convenience during development, it did come in handy to avoid typos with my LSP.